README.md

```diff
@@ -10,23 +10,26 @@ composer require markhuot/craft-pest --dev
 Handles the setup and installation of [Pest](https://pestphp.com) in to [Craft CMS](https://craftcms.com). This allows you to write tests that look something like this!
 
 ```php
+<?php
+
+use function markhuot\craftpest\helpers\http\get;
+
 it('loads the homepage')
     ->get('/')
     ->assertOk();
 
 it('has a welcoming h1 element')
-    ->expect(fn() => $this->get('/'))
-    ->querySelector('h1')->text->toBe('Welcome');
-
+    ->expect(get('/')->querySelector('h1')->text)
```

Comment:
You’re correct, but I added a (possibly) cleaner way to do this as well,

https://github.com/markhuot/craft-pest/pull/5/files#diff-e05f0d3db18081cd4c848e2b06b7aa921b43bb54f8d92b4be707290022a0a57bR26

The effect is that you can call `->expect()` on a response or a selector query to get the expectation. This way you don’t have to awkwardly wrap multiple lines in an `expect()` function call.

```php
get(‘/)
  ->assertOk()
  ->querySelector(‘.news .news__title’)
  ->expect()
  ->text->toBe([‘one’, ‘two’])
  ->count->toBe(4);
```

Reply:
I made these updates directly on your branch, hopefully that is okay?

Reply:
Of course!!